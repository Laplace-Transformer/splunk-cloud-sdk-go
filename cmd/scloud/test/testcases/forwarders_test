forwarders add-certificate --input-datafile testcases/test_forwarders_pem.pem
forwarders list-certificates
forwarders delete-certificate --slot 1
forwarders delete-certificates
