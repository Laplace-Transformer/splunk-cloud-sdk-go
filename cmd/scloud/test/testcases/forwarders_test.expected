#testcase: forwarders add-certificate --input-datafile testcases/test_forwarders_pem.pem
REQUEST URL:forwarders/v2beta1/certificates
REQUEST BODY:{{"pem":"-----BEGIN CERTIFICATE-----\nMIIC1TCCAb2gAwIBAgIJAIOAwz2qyz1FMA0GCSqGSIb3DQEBCwUAMBoxGDAWBgNV\nBAMMD3d3dy5leGFtcGxlLmNvbTAeFw0yMTA4MjcxOTAwNTVaFw0yNjA4MjYxOTAw\nNTVaMBoxGDAWBgNVBAMMD3d3dy5leGFtcGxlLmNvbTCCASIwDQYJKoZIhvcNAQEB\nBQADggEPADCCAQoCggEBANHY9Uc+AJL17z6GkJ9VoiPU0TDF3+rVjp2CLxcc2Upt\nBgX//F+MFNwY6bK3HEXE3AQPvuOtp7FQQYX+2BgLQwMiyKThKg5V4KcmsDR3zvqQ\nRdrd4lXm5KAt8kLSF+VFFO+Fm0eJG7fQgERsBFuyHg16511dh1JC2cAu6E2IaFw2\nnUWyYsvjKCl3hcPqvmVl8MIFexhPw7cyUlq68PyEgYpDXzfgE1DIiu1BGQ5z/UZ2\nXzBTQKg7/+iVfdMjftYbRqLk3MKwaM0yuohVBLkkBKuY8H+93G4qizYGAs+Ae5Hi\nEYPUkkmQMYfOXhSF7cV2aDPcCAY9oJWDaQNBk3rx4v8CAwEAAaMeMBwwGgYDVR0R\nBBMwEYIPd3d3LmV4YW1wbGUuY29tMA0GCSqGSIb3DQEBCwUAA4IBAQC5mqAoFR1t\ntYR558EYxn91VVmFFeUdXtAbkWQ6LLLQPTbWz8bQW5qoj9mYF7r2AAKnJvJoAtUX\nZYfVlHhEenGG9x8U/he/4L8IubHySMrksmoGVC5vS/0ecSD0pjObcNa6ZZH+ELbf\nO1Fm1vP/QzOZeQoH2C4tdtDNXS9JV0F4ZGOHQALEBNkO5CfOVXd3YhmGGLFxkgjs\nI135CtslJTR3+GpPHg44/Lo7VvwuSp0gJIzgLayM8Hcb7fKpZ0D2FsRkc4dDIwuR\nwDYojnaUIAuni1Dd8oguYvm5+S56XOOO9BNDorxNzqqHuwEsqszG86VBEkMAB5v+\nAQ86ecyUH90A\n-----END CERTIFICATE-----\n"}}

#testcase: forwarders list-certificates
REQUEST URL:forwarders/v2beta1/certificates
REQUEST BODY:<nil>

#testcase: forwarders delete-certificate --slot 1
REQUEST URL:forwarders/v2beta1/certificates/1
REQUEST BODY:<nil>

#testcase: forwarders delete-certificates
REQUEST URL:forwarders/v2beta1/certificates
REQUEST BODY:<nil>

